packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5697,6 +5703,7 @@ export default class Meeting extends StatelessWebexPlugin {
   };
 
   /**
+   * Question: Should we change anything in this method?
```

Comment:
Yeah, we'll need to make some changes here. Essentially, this is where we can activate and deactivate send slots on the fly. Probably makes sense to include this in the send slot manager changes.

Reply:
Thanks Bryce. In that case, will keep this as TODO